recipe/meta.yaml

```diff
@@ -1,32 +1,42 @@
-{% set name = "wxPython" %}
-{% set version = "4.0.0a2" %}
-{% set sha256 = "3acd5160d69f56b588476d1907eff67b53ad58fb2eba4dedabcdc3490d170338" %}
+{% set version = "4.0.0a3" %}
 
 package:
-  name: {{ name|lower }}
+  name: wxpython
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  fn: wxpython-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/w/wxpython/wxPython-{{ version }}.tar.gz
+  sha256: b147b1a3b4d10da88100fc89a3c4225ba6ff1f89e5da77218989b198ab92b2e6
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: True  # [unix]
+  skip: True  # [osx]
+  script: python setup.py install --single-version-externally-managed --record record.txt  # [win]
 
 requirements:
   build:
     - python
     - setuptools
+    - gstreamer  # [linux]
+    - gst-plugins-base  # [linux]
+    - mesa  # [linux]
+    - libglu  # [linux]
   run:
     - python
+    - python.app  # [osx]
     - six
+    - gstreamer  # [linux]
+    - gst-plugins-base  # [linux]
+    - mesa  # [linux]
+    - libglu  # [linux]
 
 test:
   imports:
     - wx
+  commands:
+    - conda inspect linkages -p $PREFIX wxpython  # [not win]
+    - conda inspect objects -p $PREFIX wxpython  # [osx]
```

Comment:
The issue turned out to be the package name. That is why I don't like to jinjify everything, it makes it harder to debug things.